### PR TITLE
Independancy code changes required dmapi details in the triliovault datamover conf file

### PR DESCRIPTION
**Pull request by trilio-shiwanand**

### CODE REVIEW

File: kolla-ansible/ansible/roles/triliovault/templates/triliovault-datamover.conf.j2 [hunk 1]
```diff
@@ -61,4 +67,7 @@ helper_command = sudo /usr/bin/privsep-helper
 {% endif %}
 
 [conductor]
-use_local = True
\ No newline at end of file
+use_local = True
+
+[oslo_messaging_rabbit]
+ssl = False
```
Comment (shyam-biradar) [inline kolla-ansible/ansible/roles/triliovault/templates/triliovault-datamover.conf.j2:73]: @trilio-shiwanand : Do not hardcode ssl=False. Check if kolla ansible has variable for it.

**Review by shyam-biradar — COMMENTED** [review]
Check my comments.

**Comment by trilio-shiwanand** [comment]
Added

**Comment by shyam-biradar** [comment]
Not required in master

**Pull request closed without merging**